src/lib/ngsiNotify/Notifier.h

```diff
@@ -48,6 +48,8 @@ class Notifier
   virtual void sendNotifyContextRequest(NotifyContextRequest&            ncr,
                                         const ngsiv2::Notification&      notification,
                                         const std::string&               tenant,
+					long long                        maxFailsLimit,
+					long long                        failsCounter,
```

Comment:
Review indent, please. Be careful with tabs, use only space.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/6b68a8c9ded45dd2e30c033f4d5000cddfab19de